packages/arb-token-bridge-ui/src/components/common/Layout.tsx

```diff
@@ -26,31 +26,6 @@ function Moon() {
   )
 }
 
-export const DAOBanner = () => {
```

Comment:
Are we removing the Site-Banner in this PR? Confirmed with PM?

Reply:
please refer to this comment
https://github.com/OffchainLabs/arbitrum-token-bridge/issues/994#issuecomment-1632808238